packages/react-reconciler/src/__tests__/useMemoCache-test.js

```diff
@@ -0,0 +1,367 @@
+let React;
+let ReactNoop;
+let act;
+let useState;
+let useMemoCache;
+let ErrorBoundary;
+
+describe('ReactCache', () => {
```

Comment:
Nit: "cache" is a bit overloaded at this point :D Maybe ReactForgetCache or something

Reply:
oh lol this is just copypasta from the ReactCache test, i'll update